superset-frontend/cypress-base/cypress/integration/explore/visualizations/treemap.test.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+describe('Visualization > Treemap', () => {
+  const TREEMAP_FORM_DATA = {
+    datasource: '2__table',
+    viz_type: 'treemap',
+    slice_id: 50,
+    granularity_sqla: 'year',
+    time_grain_sqla: 'P1D',
+    time_range: 'No filter',
+    metrics: ['sum__SP_POP_TOTL'],
+    adhoc_filters: [],
+    groupby: ['country_code'],
+    row_limit: 50000,
+    color_scheme: 'bnbColors',
+    treemap_ratio: 1.618033988749895,
+    number_format: '.3s',
+  };
+
+  const level0 = '.chart-container rect[style="fill: rgb(255, 90, 95);"]';
+  const level1 = '.chart-container rect[style="fill: rgb(123, 0, 81);"]';
+  const level2 = '.chart-container rect[style="fill: rgb(0, 122, 135);"]';
```

Comment:
Use color to detect lookup different levels of treemap rects. The tests might break if default color palettes updates, which is not ideal, but this is the only way I could find to find rect of different levels. The classnames used in previous tests have stopped working:

```js
      cy.get('.chart-container svg rect.parent').should('have.length', 7);
      cy.get('.chart-container svg rect.child').should('have.length', 214);
```